src/features/services/components/data-settings.tsx

```diff
@@ -0,0 +1,27 @@
+import { Box, Button, Flex, useDisclosure, Center } from "@liftedinit/ui";
+import { PutValueModal } from "../components";
+import { useAccountsStore } from "features/accounts";
+import { ANON_IDENTITY } from "@liftedinit/many-js";
+
+export function DataSettings() {
+  const account = useAccountsStore((s) => s.byId.get(s.activeId));
+  const { isOpen, onOpen, onClose } = useDisclosure();
+  return (
+    <>
+      <Box p={6} bg="white" mt={9} boxShadow="xl">
+        <Center>
+          <pre>PLACEHOLDER FOR liftedinit/roadmap#17</pre>
+        </Center>
```

Comment:
@mjsorribas Your table can go here. I'm trying to preemptively avoid merge conflicts by leaving space.